src/app/personside/infotabs/dyplenkeTest/utils.ts

```diff
@@ -3,3 +3,8 @@ import { ReactWrapper } from 'enzyme';
 export function getAktivTab(infoTabs: ReactWrapper) {
     return infoTabs.find('button[role="tab"][aria-selected=true]').text();
 }
+
+export const utbetalingerTest = {
```

Comment:
Hva er best practice her? Lage en _const_ for hver av gruppene (utbetalinger, ytelser ...) eller alle i samme variabel med lengre navn (_utbetalingerOversikt_ )?

Reply:
Kan evt gå for å bruk en enum.

Reply:
Hm.. Har ikke noe riktig svar, må vel finne krysningspunktet mellom hva som er lettest å lese og hva som gir robust kode. Tenker at hver test godt sikkert kan ha sin egen const.. :-)

Reply:
Enig der! For å få det mest ryddig